### PR TITLE
Approval checkbox required by default

### PR DESCRIPTION
**Pull request by pavi41**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

**Review by rismehta — CHANGES_REQUESTED** [review]
Please add a runtime test case for the same, no PR's without test case.

File: examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/termsandconditions/_cq_template/.content.xml [hunk 1]
```diff
@@ -19,6 +19,7 @@
     jcr:title="I agree to the terms &amp; conditions"
     sling:resourceType="core/fd/components/form/checkbox/v1/checkbox"
     enabled="false"
+    required="true"
```
Comment (barshat7) [inline examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/termsandconditions/_cq_template/.content.xml:22]: Please add the changes in archetype proxy also
https://github.com/adobe/aem-project-archetype/blob/develop/src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/termsandconditions/_cq_template.xml

**Comment by pavi41** [comment]
> Please add a runtime test case for the same, no PR's without test case.

Added test case for the same!

**Review by rismehta — APPROVED** [review]

**Review by rismehta — CHANGES_REQUESTED** [review]
You should make it required here, https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/termsandconditions/v1/termsandconditions/_cq_template.xml and also in the archetype

**Comment by pavi41** [comment]
> You should make it required here, https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/termsandconditions/v1/termsandconditions/_cq_template.xml and also in the archetype

@rismehta I have added it in the file you mentioned, will create the seperate PR for archetype

**Review by rismehta — APPROVED** [review]

**Pull request merged**